printer/src/main/java/org/odk/collect/printer/HtmlPrinter.kt

```diff
@@ -0,0 +1,41 @@
+package org.odk.collect.printer
+
+import android.content.Context
+import android.print.PrintAttributes
+import android.print.PrintManager
+import android.webkit.WebResourceRequest
+import android.webkit.WebView
+import android.webkit.WebViewClient
+
+class HtmlPrinter {
```

Comment:
I was in two minds about whether this should be tested or not, but it feels to me like anything with `WebView` is going to be hard to write tests around we'd feel confident about and I feel like you've found a good "edge" here to contain the untested stuff.